posthog/api/test/test_dashboard.py

```diff
@@ -8,13 +8,32 @@
 
 from posthog.models import Dashboard, Filter, Insight, Team, User
 from posthog.models.organization import Organization
-from posthog.test.base import APIBaseTest
+from posthog.test.base import APIBaseTest, QueryMatchingTest, snapshot_postgres_queries
 from posthog.utils import generate_cache_key
 
 
-class TestDashboard(APIBaseTest):
+class TestDashboard(APIBaseTest, QueryMatchingTest):
     CLASS_DATA_LEVEL_SETUP = False
 
+    @snapshot_postgres_queries
+    def test_retrieve_dashboard_list(self):
```

Comment:
Can we specifically add a test that checks the number of queries is independent of number of rows, this way we don’t have to be concerned about the specific number just that it scales as expected